anillo/middlewares/cors.py

```diff
@@ -0,0 +1,23 @@
+import functools
+
+def wrap_cors_response(func=None, *, allow_origin='*', allow_headers='Origin, X-Requested-With, Content-Type, Accept'):
```

Comment:
I agree with @niwinz , probably the best aproach is a set or a tuple, the length of the line is less important, but you can write it as:
```
def wrap_cors_response(func=None, *, allow_origin='*',
                       allow_headers=('Origin', 'X-Requested-With',
                                      'Content-Type', 'Accept')):
```